operate/client/src/modules/components/TabView/styled.tsx

```diff
@@ -30,6 +30,12 @@ const Tab = styled(BaseTab)`
   padding: 9px var(--cds-spacing-05) var(--cds-spacing-03) !important;
 `;
 
+const TabInternalContainer = styled.div`
+  display: flex;
+  alignitems: center;
+  gap: 8px;
+`;
```

Comment:
We should use a horizontal [stack](https://react.carbondesignsystem.com/?path=/docs/layout-stack--overview) instead. With a spacing from the [spacing scale](https://carbondesignsystem.com/elements/spacing/overview/) (`$spacing-03` in this case).